server/src/main/kotlin/io/provenance/objectstore/gateway/server/ObjectStoreGatewayServer.kt

```diff
@@ -39,30 +38,24 @@ class ObjectStoreGatewayServer(
         request: GatewayOuterClass.PutObjectRequest,
         responseObserver: StreamObserver<GatewayOuterClass.PutObjectResponse>
     ) {
-        objectService.putObject(request.objectBytes.toByteArray(), request.type.takeIf { it.isNotBlank() }, publicKey()).let {
-        responseObserver.onNext(
-            GatewayOuterClass.PutObjectResponse.newBuilder()
-                .setHash(it)
-                .build()
-        )
-    }
+        objectService.putObject(request.`object`, publicKey()).let {
```

Comment:
RIP in reserved words from proto.  IIRC if something gets named `final` things get super weird in Java lol

Reply:
yeah, I thought about changing the field name... but wasn't coming up with anything that sounded good to me